nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2645,6 +2645,15 @@ documentation of the proxy for guidance for your deployment environment and use
 ...
 ....
 
+* If NiFi is running securely, configuration must be updated to allow expected Host and context paths HTTP headers.
+
+** By default, NiFi will only accept HTTP requests with a Host header matching the host[:port] that it is bound to. If NiFi is to accept requests directed to a different
+host[:port] the expected values need to be configured. This may be required when running behind a proxy or in a containerized environment. This is configured in a comma
+separated list in _nifi.properties_ using the `nifi.web.proxy.host` property. (e.g. localhost:18443, proxyhost:443)
```

Comment:
The part about host headers is accurate -- it is only activated when running securely, and this is the correct property to set.

Reply:
I would also add a note here that IPv6 addresses can be tricky, and [RFC 5952 Sections 4](https://tools.ietf.org/html/rfc5952#section-4) and [6](https://tools.ietf.org/html/rfc5952#section-6) are recommended reading